### DISCUSSION TITLE
Split screen of tabs in one place

### DISCUSSION THREAD

**Discussion by ghost** [minbrowser/min#1573] [Ideas]
Hi! 
Had an idea on split tabs, like having 2 tabs on one view to like compare sites or something. It saves the hassle of going from tab to tab and on that split tab screen you could have like a resize page width tool if some pages don't get along.
I think it would be neat with fullscreen and focus mode, so there won't be like window manager to distract or misclick something.

Let me know if it's a good idea.

**Comment by blackgwe** [comment]
It reminds me on tmux panes → https://i.ytimg.com/vi/BHhA_ZKjyxo/maxresdefault.jpg

**Reply by ghost** [reply]
Yes, something like that!
